fusestore/store/src/meta_service/raftmeta.rs

```diff
@@ -159,6 +167,7 @@ impl MetaStore {
             log,
             sm,
             hs,
+            snapshot_idx: Arc::new(Mutex::new(0)),
```

Comment:
Why use `0` instead of `current_snapshot.snapshot_idx` here?

Reply:
My mistake! Thanks for saving my ass!

Reply:
There is no such field `snapshot_idx` in it. Only `snapshot_id`.

The index being reset is not big deal with in-memory impl, for now.
In the future all of the data should be loaded from disk. This part should be refined.

I'm gonna add a TODO for it.

Thank you my man!

Reply:
Got it, rest LGTM.